src/main/resources/api.yaml

```diff
@@ -396,6 +402,36 @@ paths:
               schema:
                 $ref: '#/components/schemas/SprintSummaryDto'
```

Comment:
O tu https://github.com/papaya-agh/papaya-interface/pull/19 się pojawia `estimatedTimePlanned` i `finalTimePlanned`  (oraz `timeBurned`, które już było) w `SprintDto` - a to właśnie one mają być brane z Jiry, także wydaje mi się, że tak najlepiej to zrobić